tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,41 @@
+# setup
+import great_expectations as gx
+import great_expectations.expectations as gxe
+from great_expectations.core.expectation_suite import ExpectationSuite
+
+context = gx.get_context()
+suite_name = "test-suite"
+
+# create a suite
+suite = context.suites.add(suite=ExpectationSuite(name=suite_name))
+
+# get a suite
+suite = context.suites.get(name=suite_name)
+
+# add expectations
+suite.add_expectation(
+    gxe.ExpectColumnValuesToBeInSet(column="passenger_count", value_set=[1, 2, 3, 4, 5])
+)
+suite.add_expectation(gxe.ExpectColumnValuesToNotBeNull(column="pickup_datetime"))
+
+# find a single expectation
+expectation = next(
+    expectation
+    for expectation in suite.expectations
+    if isinstance(expectation, gxe.ExpectColumnValuesToNotBeNull)
+)
+
+# update a single expectation
+expectation.column = "pickup_location_id"
+expectation.save()
+
+# update multiple expectations as a batch
+for expectation in suite.expectations:
+    expectation.notes = "This Expectation was generated as part of GX Documentation."
+suite.save()
+
+# delete an expectation
+suite.delete_expectation(expectation=expectation)
```

Comment:
Can we move this up to the part about finding an expectation so it's not relying on the left over iteration from above?

Reply:
okay - the reason its at the bottom is that the two calls above are also relying on the `expectation` variable retrieved above. Should we include `get` code with each example?

Reply:
It would be best if each example is as self-contained as possible.  So I would vote for including the `get` code each time, unless the examples are grouped very closely and are related to the same process.